src/components/MainNavigation/MainNavigationChildItem.vue

```diff
@@ -1,14 +1,16 @@
 <template>
-  <li
-    :class="[
-      'w-full no-underline my-1 max-h-12 flex items-center font-light text-sm text-left text-gray-800 hover:text-primary-700',
-      { 'font-medium bg-white-300 rounded-l-full ring-2 ring-primary-100': active }
-    ]"
-    data-testid="nav-child-item"
-  >
+  <li class="w-full my-1 max-h-12 hover:bg-gray-50 relative">
+    <div
+      v-if="isActive"
+      class="absolute top-2 -left-[3px] w-1 h-5 bg-gray-800 rounded-full"
+    />
     <LobLink
       :to="to"
-      class="block w-44 rounded-l-full py-2 pl-4 !text-sm whitespace-nowrap overflow-hidden !text-gray-800 focus:outline-none focus:ring-0 focus-visible:ring-2 focus-visible:ring-primary-100"
+      :class="[
+        'block !w-full py-2 pl-4 !type-small-500 whitespace-nowrap  !text-gray-500',
+        'focus:ring-0 focus-visible:bg-gray-50 focus-visible:!ring-0 focus-visible:!rounded-none focus-visible:!ring-offset-0',
+        { '!text-gray-800 !type-small-600': isActive }
```

Comment:
nit: do we need the `!` in `!text-gray-500` and `!text-gray-800`? 
if not, maybe we can do something like `{ isActive ? 'text-gray-800' : 'text-gray-500' }` (in general, I prefer avoiding overwriting classes by using `!` if possible)

Reply:
thanks Juan, I added this change for both Item and ChildItem classes